### ISSUE TITLE
Please add support for Pinterest

### ISSUE THREAD

**Issue by Slessi** [nraboy/ng-cordova-oauth#199]
Pinterest implements OAuth2 - https://developers.pinterest.com/docs/api/overview/#authentication

Would be nice to have :)

Edit: Made a pull request for it #201

**Comment by nraboy** [comment]
Waiting on confirmation of https://github.com/nraboy/ng-cordova-oauth/pull/201 before merge.

**Comment by nraboy** [comment]
Closing this because I feel Pinterest has some questionable behavior in their flow API.  More described in the https://github.com/nraboy/ng-cordova-oauth/pull/201 PR.
